spec-0014/index.md

```diff
@@ -0,0 +1,371 @@
+---
+title: "SPEC 14 — Interactive Documentation"
+shortcutDepth: 3
+number: 14
+# yyyy-mm-dd, gotta specify a date so that it's parseable
+date: 2025-04-14
+author:
+# <!-- Please check your emails -->
+  - "Agriya Khetarpal <agriyakhetarpal@outlook.com>"
+  - "Albert Steppi <albert.steppi@gmail.com>"
+  - "Matthias Bussionnier <bussonniermatthias@gmail.com>"
+  - "Melissa Weber Mendonça <melissawm@gmail.com>"
+  - "Ralf Gommers <ralf.gommers@gmail.com>"
+# <!-- Maybe Loïc Estève and Henry Schreiner too? Robert Hood Chatham and Gyeongjae Choi as well perhaps, they'll be reviewing this when this is public anyway and any others I missed? -->
+discussion: https://discuss.scientific-python.org/notopenedyet, will link here when we make things public
+# <!-- Also to keep in mind, this SPEC closes the following issues:
+# https://github.com/scientific-python/summit-2024/issues/19
+# and https://github.com/Quansight-Labs/czi-scientific-python-mgmt/issues/19
+# and https://github.com/scientific-python/cookie/issues/157 -->
+is-draft: true
+endorsed-by:
+# TODO: ask for formal approval from these projects that adopted interactive docs after SPEC review
+  - scikit-learn
+  - scipy
+  - numpy
+  - scikit-image # soon
+# SymPy is not among the ore projects but has been in the discussion to be included – it needs approval from SymPy's side. see https://discuss.scientific-python.org/t/spec-0-include-sympy/975
+  - pywavelets # not a core project but the first one to get it done so it serves as an inspiration
+  - sympy # soon
+  - pandas # soon
+  - statsmodels # soon
+---
+
+## Description
+
+<!--
+Briefly and clearly describe the recommendation.
+-->
+
+A common pain point for new users coming to Scientific Python projects—particularly projects containing compiled code—has been their difficulty of installation, with historic package management troubles arising as a by-product of Python's strong capabilities for wrapping low-level languages such as C, C++, Fortran, and Rust. While the situation has improved with enhancements across the Python package management ecosystem, setting up a working Python environment can still be quite difficult for beginners. Even for experienced Python programmers, it can take a nontrivial amount of time to correctly install an unfamiliar package.
+
+In-browser interactive documentation offers users a chance to try out and explore projects without taking effort to install them locally. Even when users are able to install projects, interactive documentation provides a more immersive experience than copying and pasting code snippets into a local REPL, IDE/editor or Notebook. It can thus offer benefits for beginners and more experienced programmers alike.
+
```

Comment:
I think it would be good to have a bridge paragraph here, clearly and concretely describing what the interactive documentation we're proposing looks like, without getting into technical details and without attempting to write in a very precise legal way; so readers can get in their minds what we're proposing right away.

Reply:
I added two screenshots via d055d56bbf37f39f728ef565aa86e1c4d10cae1e, one with the button as is and another when it's opened.